llvm/test/CodeGen/AMDGPU/llc-pipeline-npm.ll

```diff
@@ -0,0 +1,144 @@
+; UNSUPPORTED: expensive_checks
```

Comment:
Why? Is this just from the verifier being on by default? You can use -verify-machineinstrs=0 (and doesn't the -disable-verify just skip the IR verifier on the input)?